recipes/zfp/bld.bat

```diff
@@ -0,0 +1,17 @@
+setlocal EnableDelayedExpansion
+
+:: Make a build folder and change to it.
+mkdir build
+cd build
+
+:: Configure using the CMakeFiles
+cmake -DBUILD_ZFPY=ON -DZFP_WITH_OPENMP=OFF -DCMAKE_INSTALL_PREFIX=${PREFIX} ..
```

Comment:
Try using `-GNinja` and add `ninja` to the requirements. Doesn't solve all your problems but at least may solve some problems.

Reply:
Will do.  Thanks!